### DISCUSSION TITLE
wails v2 for linux?

### DISCUSSION THREAD

**Discussion by zakaria-chahboun** [wailsapp/wails#1145] [Q&A]
Hi guys! how i can try the v2 on Linux? and is there a specific date for support Linux on the [website](https://wails.io/)
Thanks :smiling_face_with_three_hearts:

**Comment by leaanthony** [comment]
You can use the experimental build by installing the bleeding edge then adding `-tags exp` to all the wails commands. https://wails.io/docs/guides/bleeding-edge

**Reply by zakaria-chahboun** [reply]
it gives me this
```
Wails CLI v2.0.0-beta.30

ERROR: Linux version coming soon!

```
